npc/jobs/1-1e/gunslinger.txt

```diff
@@ -183,10 +183,16 @@ que_ng,152,167,3	script	Master Miller	901,{
 			callfunc "Job_Change",Job_Gunslinger;
 			set GUNS_Q,6;
 			completequest 6024;
-			if (rand(1,2) == 1) {
-				getitem 13100,1; // Six_Shooter
+			if(checkre(0) == 1){
```

Comment:
```suggestion
			if (checkre(0)) {
```

Reply:
I don't know why everyone is so fond of adding extra spaces, but I'll apply the suggestion because we don't really need the comparative.